services/UserService.js

```diff
@@ -7,6 +9,21 @@ export default class UserService extends BaseService {
     this.userModel = userModel;
   }
 
+  async login(login, password) {
+    try {
+      const user = await this.userModel.findOne({
+        where: { login, password, isDeleted: false },
+      });
+      if (user) {
+        const payload = { id: user.id, login: user.login };
+        return jwt.sign(payload, config.secret, { expiresIn: 3600 });
```

Comment:
To handle expiry you need to refresh your JWTs else user have to login again and again.
Check how the refreshing token is handled in JWT.